nautobot/extras/templates/extras/role.html

```diff
@@ -0,0 +1,52 @@
+{% extends 'generic/object_detail.html' %}
+{% load helpers %}
+
+{% block content_left_page %}
+            <div class="panel panel-default">
+                <div class="panel-heading">
+                    <strong>Role</strong>
+                </div>
+                <table class="table table-hover panel-body attr-table">
+                    <tr>
+                        <td>Name</td>
+                        <td>
+                            {{ object.name }}
+                        </td>
+                    </tr>
+                    <tr>
+                        <td>Slug</td>
+                        <td>
+                            {{ object.slug }}
+                        </td>
+                    </tr>
+                    <tr>
+                        <td>Weight</td>
+                        <td>
+                            {{ object.weight }}
```

Comment:
Since `weight` is nullable:

```suggestion
                            {{ object.weight|placeholder }}
```